app/views/widgets/_smart_banner.html.slim

```diff
@@ -0,0 +1,13 @@
+- cookie = cookies[:smart_banner_dismiss]
+- if (browser.device.mobile? || browser.device.tablet?) && (cookie.nil? || (Time.now >= Time.strptime(cookie, "%FT%R") + 15.days))
+  .smart-banner
+    .info
+      i.icon-close
+      a href="#{browser.platform.android? ? android_store_page : apple_store_page}"
+        = image_tag "icon", class: "app-icon"
+      h3 Baixe o app da Mudamos
+  javascript:
+    $(".smart-banner .info i.icon-close").click(function() {
+      $(".smart-banner").slideUp();
+      document.cookie = "smart_banner_dismiss=" + moment().format();
```

Comment:
Using a simple timestamp is way easier on this case. On your JS side, you simple call `Date.now()`, and on the Ruby side you simple call `Time.at` passing the timestamp.